aws-amplify/docs
# Update husky to fix dependabot issue 25

_Issue #, if available:_ https://github.com/aws-amplify/docs/security/dependabot/25

_Description of changes:_
- Update husky to `^8.0.1` so there isn't a dependency for `semver-regex`
- Update cspell to so that it works with this version of husky 

_To test:_
1. Clone branch
2. Run `yarn install` in `/docs` directory
3. Edit an MDX file with a typo
4. Stage the file and try to commit it
   - Verify that commit is unsuccessful because there is a typo

![image](https://user-images.githubusercontent.com/54393192/179268206-3fe9b96e-a69f-408d-a3d7-660400c766aa.png)


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
